ibis/backends/base/__init__.py

```diff
@@ -223,6 +224,8 @@ def _ipython_key_completions_(self) -> list[str]:
 
 
 class _FileIOHandler:
+    pandas_converter = PandasData
```

Comment:
Slight preference for this to be a private attribute if we keep this, just so it doesn't show up in tab-completion for users.

Reply:
Removed the attribute entirely.